tests/BinanceSmartChain/SignerTests.cpp

```diff
@@ -44,7 +44,7 @@ TEST(BinanceSmartChain, SignNativeTransfer) {
 
     // addr: 0xB9F5771C27664bF2282D98E09D7F50cEc7cB01a7  mnemonic: isolate dismiss ... cruel note
     auto privateKey = PrivateKey(parse_hex("4f96ed80e9a7555a6f74b3d658afdd9c756b0a40d4ca30c42c2039eb449bb904"));
-    auto signer = SignerExposed(TWEthereumChainIDBinanceSmartChain);
+    auto signer = SignerExposed(97);
```

Comment:
should we switch test case to `TWEthereumChainIDBinanceSmartChain` ?

Reply:
no need, we already have mainnet test case in the app